app/src/main/java/com/jerboa/ui/components/common/InputFields.kt

```diff
@@ -679,17 +676,10 @@ fun PreviewLines(
 fun MyMarkdownText(
     markdown: String,
     color: Color = MaterialTheme.colorScheme.onSurface,
-    onClick: () -> Unit,
+    onClick: (() -> Unit)? = null,
+    onLongClick: (() -> Unit)? = null,
 ) {
-    MarkdownText(
-        markdown = markdown,
-        modifier = Modifier.fillMaxSize(),
-        onClick = onClick,
-        color = color,
-        fontSize = MaterialTheme.typography.bodyLarge.fontSize.times(MARKDOWN_FONT_MULTIPLIER),
-//        style = MaterialTheme.typography.titleLarge,
-//        imageLoader =  LocalImageLoader.current
-    )
+    MarkdownHelper.CreateMarkdownView(markdown = markdown, color = color, onClick = onClick, onLongClick = onLongClick)
```

Comment:
Hopefully this is failing lint. Run `./gradlew formatKotlin`.

Reply:
This was not being caught by `formatKotlin`.  Updated now